r/vignettes/install.Rmd

```diff
@@ -157,11 +329,27 @@ something other than the RSPM binary mirror URLs).
 * Download the dependency files (`cpp/thirdparty/download_dependencies.sh` may be helpful)
 * Copy the directory of dependencies to the offline computer
 * Create the environment variable `ARROW_THIRDPARTY_DEPENDENCY_DIR` on the offline computer, pointing to the copied directory.
-* Install the `arrow` package as usual.
+* Install the arrow package as usual.
+
+# Installing libarrow dependencies
```

Comment:
I think this section should probably be further up. Especially the S3 dependencies are needed if you use a binary or not, so we should flag those up there before going into more detail about building from source.

Reply:
Moved to the top